src/core/ready.js

```diff
@@ -5,12 +5,13 @@ define( [
 ], function( jQuery, document ) {
 
 // The deferred used on DOM ready
-var readyList;
+var readyList = jQuery.Deferred(),
+	readyPromise = readyList.promise();
```

Comment:
I'm pretty sure you can get rid of `readyPromise` and just use `readyList` directly.